src/unistd.rs

```diff
@@ -760,11 +760,13 @@ pub fn execvpe(filename: &CString, args: &[CString], env: &[CString]) -> Result<
 ///
 /// This function is similar to `execve`, except that the program to be executed
 /// is referenced as a file descriptor instead of a path.
+///
+/// Note for NetBSD and OpenBSD: although rust-lang/libc includes it (under
```

Comment:
I'd change these to regular comments (only two slashes), as they don't need to be in the docs for it.

Also, since we're technically removing a function, we should have a CHANGELOG entry indicating this.